### PR TITLE
MAUI-856989-Add Accessibility For All Picker

### PR DESCRIPTION
**Pull request by Sathish-087**
**Description**
Add accessibility for all picker control.

Content Review Task Link: https://dev.azure.com/Syncfusion-General/Documentation/_workitems/edit/763279/

### CODE REVIEW

File: MAUI/DateTimePicker/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+documentation: ug
+---
+
+# Accessibility in .NET MAUI Date Time Picker (SfDateTimePicker)
+
+The .NET MAUI Date Time Picker control has interact with the `Header`, `Column Header`, `Footer`, and `Picker Items`.
+
+## Header Layout
+
+The SfDateTimePicker control has intract header text with localization of header layout. It contains DateFormat and TimeFormat property of the DateTimePickerHeaderView. For example,
```
Comment (M-A32) [inline MAUI/DateTimePicker/accessibility.md:16]: The SfDateTimePicker control has interact header text with localization of header layout. It contains DateFormat and TimeFormat property of the DateTimePickerHeaderView. For example,

File: MAUI/DateTimePicker/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+</tr>
+<tr>
+<td>Time Format</td>
+<td>10:15:30 AM (or) PM</td>
+</tr>
+</table>
+
+## Column Header Layout
+
+The SfDateTimePicker control has intract column header text with localization of column header layout. It contains DayHeaderText, MonthHeaderText, YearHeaderText, HourHeaderText, MinuteHeaderText, SecondHeaderText, and MeridiemHeaderText properties of the DateTimePickerColumnHeaderView. The default value of the DayHeaderText property is "Day", MonthHeaderText is "Month", YearHeaderText is "Year", HourHeaderText property is "Hour", MinuteHeaderText is "Minute", SecondHeaderText is "Second", and MerdiemHeaderText is "string.Empty".
```
Comment (M-A32) [inline MAUI/DateTimePicker/accessibility.md:35]: The SfDateTimePicker control has interact column header text with localization of column header layout. It contains DayHeaderText, MonthHeaderText, YearHeaderText, HourHeaderText, MinuteHeaderText, SecondHeaderText, and the MeridiemHeaderText properties of the DateTimePickerColumnHeaderView. The default value of the DayHeaderText property is "Day", MonthHeaderText is "Month", YearHeaderText is "Year", HourHeaderText property is "Hour", MinuteHeaderText is "Minute", SecondHeaderText is "Second", and MerdiemHeaderText is "string.Empty".

File: MAUI/DateTimePicker/accessibility.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## Footer Layout
+
+The SfDateTimePicker control has intract validation buttons (OK and Cancel) with localization of footer layout. The Default value of the OkButtonText property is "OK", and CancelButtonText is "Cancel".
```
Comment (M-A32) [inline MAUI/DateTimePicker/accessibility.md:39]: The SfDateTimePicker control has interact validation buttons (OK and Cancel) with localization of footer layout. The Default value of the OkButtonText property is "OK", and CancelButtonText is "Cancel".

File: MAUI/DateTimePicker/accessibility.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+## Picker Items
+
+The SfDateTimePicker control has intract with item source in picker items. The item source contains based on Date format. For example,
```
Comment (M-A32) [inline MAUI/DateTimePicker/accessibility.md:43]: The SfDateTimePicker control has interact with item source in picker items. The item source contains based on Date format. For example,

File: MAUI/Picker/accessibility.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+documentation: ug
+---
+
+# Accessibility in .NET MAUI Picker (SfPicker)
+
+The .NET MAUI Picker control has interact with the `Header`, `Column Header`, `Footer`, and `Picker Items`.
+
+## Header Layout
+
+The SfPicker control has intract header text with localization of header layout.
```
Comment (M-A32) [inline MAUI/Picker/accessibility.md:16]: The SfPicker control has interact header text with localization of header layout.

File: MAUI/Picker/accessibility.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+## Column Header Layout
+
+The SfPicker control has intract column header text with localization of column header layout.
```
Comment (M-A32) [inline MAUI/Picker/accessibility.md:20]: The SfPicker control has interact column header text with localization of column header layout.

File: MAUI/Picker/accessibility.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+## Footer Layout
+
+The SfPicker control has intract validation buttons (OK and Cancel) by setting the OkButtonText and CancelButtonText properties of the PickerFooterView with localization of footer layout. The Default value of the OkButtonText property is "OK", and CancelButtonText is "Cancel".
```
Comment (M-A32) [inline MAUI/Picker/accessibility.md:24]: The SfPicker control has interact validation buttons (OK and Cancel) by setting the OkButtonText and CancelButtonText properties of the PickerFooterView with localization of footer layout. The Default value of the OkButtonText property is "OK", and CancelButtonText is "Cancel".

File: MAUI/TimePicker/accessibility.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+The SfTimePicker control has intract column header text with localization of column header layout. It contains HourHeaderText, MinuteHeaderText, SecondHeaderText, and MeridiemHeaderText properties of the TimePickerColumnHeaderView. The default value of the HourHeaderText property is "Hour", MinuteHeaderText is "Minute", SecondHeaderText is "Second", and MerdiemHeaderText is "string.Empty".
+
+## Footer Layout
+
+The SfTimePicker control has intract validation buttons (OK and Cancel) with localization of footer layout. The Default value of the OkButtonText property is "OK", and CancelButtonText is "Cancel".
+
+## Picker Items
+
+The SfTimePicker control has intract with item source in picker items. The item source contains based on time format. For example,
```
Comment (M-A32) [inline MAUI/TimePicker/accessibility.md:28]: The SfTimePicker control has interact with item source in picker items. The item source contains based on time format. For example,

File: MAUI/TimePicker/accessibility.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+The SfTimePicker control has intract header text with localization of header layout.
+
+## Column Header Layout
+
+The SfTimePicker control has intract column header text with localization of column header layout. It contains HourHeaderText, MinuteHeaderText, SecondHeaderText, and MeridiemHeaderText properties of the TimePickerColumnHeaderView. The default value of the HourHeaderText property is "Hour", MinuteHeaderText is "Minute", SecondHeaderText is "Second", and MerdiemHeaderText is "string.Empty".
+
+## Footer Layout
+
+The SfTimePicker control has intract validation buttons (OK and Cancel) with localization of footer layout. The Default value of the OkButtonText property is "OK", and CancelButtonText is "Cancel".
```
Comment (M-A32) [inline MAUI/TimePicker/accessibility.md:24]: The SfTimePicker control has interact validation buttons (OK and Cancel) with localization of footer layout. The Default value of the OkButtonText property is "OK", and CancelButtonText is "Cancel".

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

File: MAUI/DatePicker/accessibility.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+documentation: ug
+---
+
+# Accessibility in .NET MAUI Date Picker (SfDatePicker)
+
+The .NET MAUI Date Picker control has interact with the `Header`, `Column Header`, `Footer`, and `Picker Items`.
+
+## Header Layout
+
+The SfDatePicker control has intract header text with localization of header layout.
```
[the remark this answers is not in the document]
Comment (Sathish-087) [reply MAUI/DatePicker/accessibility.md:16]: Changed

File: MAUI/DatePicker/accessibility.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+
+The SfDatePicker control has intract column header text with localization of column header layout. It contains DayHeaderText, MonthHeaderText, and YearHeaderText properties of the DatePickerColumnHeaderView. The default value of the DayHeaderText property is "Day", MonthHeaderText is "Month", and YearHeaderText is "Year".
+
+## Footer Layout
+
+The SfDatePicker control has intract validation buttons (OK and Cancel) with localization of footer layout. The Default value of the OkButtonText property is "OK", and CancelButtonText is "Cancel".
+
+## Picker Items
+
+The SfDatePicker control has intract with item source in picker items. The item source contains based on Date format. For example,
```
[the remark this answers is not in the document]
Comment (Sathish-087) [reply MAUI/DatePicker/accessibility.md:28]: Changed

File: MAUI/DateTimePicker/accessibility.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: Accessibility with .NET MAUI Date Time Picker Control | Syncfusion
+description: Learn here all about accessibility with Syncfusion .NET MAUI Date Time Picker (SfDateTimePicker) control.
```
[the remark this answers is not in the document]
Comment (Sathish-087) [reply MAUI/DateTimePicker/accessibility.md:4]: Changed

File: MAUI/DateTimePicker/accessibility.md (hunk 1 above)
Comment (Sathish-087) [reply MAUI/DateTimePicker/accessibility.md:16]: Changed

File: MAUI/DateTimePicker/accessibility.md (hunk 2 above)
Comment (Sathish-087) [reply MAUI/DateTimePicker/accessibility.md:35]: Changed

File: MAUI/DateTimePicker/accessibility.md (hunk 3 above)
Comment (Sathish-087) [reply MAUI/DateTimePicker/accessibility.md:39]: Changed

File: MAUI/Picker/accessibility.md (hunk 5 above)
Comment (Sathish-087) [reply MAUI/Picker/accessibility.md:16]: Changed

File: MAUI/Picker/accessibility.md (hunk 6 above)
Comment (Sathish-087) [reply MAUI/Picker/accessibility.md:20]: Changed

File: MAUI/Picker/accessibility.md (hunk 7 above)
Comment (Sathish-087) [reply MAUI/Picker/accessibility.md:24]: Changed

File: MAUI/TimePicker/accessibility.md (hunk 8 above)
Comment (Sathish-087) [reply MAUI/TimePicker/accessibility.md:28]: Changed

File: MAUI/TimePicker/accessibility.md (hunk 9 above)
Comment (Sathish-087) [reply MAUI/TimePicker/accessibility.md:24]: Changed

**Review by MuniappanSubramanian — APPROVED** [review]
Changes are fine

**Pull request merged**